plugins/thunderbolt/fu-thunderbolt-controller.c

```diff
@@ -151,7 +151,7 @@ fu_thunderbolt_controller_can_update(FuThunderboltController *self)
 }
 
 static gboolean
-fu_thunderbolt_controller_set_port_offline_cb(gpointer user_data)
+fu_thunderbolt_controller_set_port_online_cb(gpointer user_data)
```

Comment:
I thought this was intentional, @krankun please confirm.

Reply:
echo 1 > offline // its a offline operation
echo 0 > offline // its online operation 

so this change looks ok to me